docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+also has blobs that are referenced in the manifest in the `layers` property. It is
+important to note that the `config` property is required in the manifest and
+in this case the config is an empty blob as per the [empty descriptor guidance][empty-descriptor].
+
+```json
+ {
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2,
+    "data": "e30="
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar",
+      "digest": "sha256:d2a84f4b8b650937ec8f73cd8be2c74add5a911ba64df27458ed8229da804a26",
+      "size": 12,
+      "annotations": {
+        "org.opencontainers.image.title": "hello.txt"
+      }
+    }
+  ],
+  "annotations": {
+    "org.opencontainers.image.created": "2023-08-03T00:21:51Z"
+  }
+}
+```
+
+### Artifacts with config
+
+Clients can use `config.mediaType` property  to
+declare the artifact type. The `config.mediaType` may be used when artifacts have a valid config blob and
+the mediaType matches the config blob and type of the artifact. For example the
+artifact type below for the helm chart is `application/vnd.cncf.helm.config.v1+json`
+
+```json
+{
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "application/vnd.cncf.helm.config.v1+json",
+    "digest": "sha256:34bf03806938a59ee7dc3e2c33e314d0eaef573313ff9dcc677113502d568523",
+    "size": 145
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.cncf.helm.chart.content.v1.tar+gzip",
+      "digest": "sha256:4d80464e9d8e9f3ba92e6ead6d3b5afd0532cb0a81e980599a0bced99fdc6e01",
+      "size": 3763
+    }
+  ]
+}
+```
+
+### Annotations only artifacts
+
+Artifacts may store metadata in the manifest as annotations and do not have a config or blobs.
+For these the `artifactType` property is used to declare the type of the artifact.
+The config property is required in the manifest for maximum compatiblity an empty layer is also created.
+as per the  [empty descriptors guidance][empty-descriptor].  
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.empty.v1+json",
+      "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+      "size": 2
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2023-01-02T03:04:05Z",
+    "com.example.info": "useful-info"
+  }
+}
+```
+
+### Deciding on artifactType or config.mediaType
```

Comment:
Does this need to be this complex? See [comment above](https://github.com/oras-project/oras-www/pull/238/files#r1287673587) Setting the `artifactType` can dramatically simplify the creation and consumption. 
- If the `artifactType` property exists, it takes precedence. Always
- If there is no `artifactType`, defer to `config.mediaType` as this is the backwards compact/fallback

Reply:
For authors this is needed.